x/programs/cmd/simulator/README.md

```diff
@@ -0,0 +1,231 @@
+# Program VM simulator
+
+## Introduction
+
+The VM simulator provides a tool for testing and interacting with HyperSDK Wasm
+`Programs`.
+
+## Build
+
+```sh
+go build
+./simulator -h
+```
+
+## Testing a HyperSDK Programs
+
+To test a HyperSDK Program you will need to create a `run` file. We currently support `run` files in both `JSON` and `YAML`.
```

Comment:
But YAML is a superset of JSON, we shouldn't need to support both 😞. 

Haha, you can resolve this, I know that no one is going to write pure json in a file and save it as a `.yaml ` instead of saving as .json